yt_dlp/extractor/twitch.py

```diff
@@ -403,6 +488,10 @@ def _real_extract(self, url):
                 })),
             vod_id, 'mp4', entry_protocol='m3u8_native')
 
+        # extract storyboard
+        if video.get('storyboard') is not None and info.get('duration') is not None:
```

Comment:
```suggestion
        if video.get('storyboard') and info.get('duration'):
```
or, even better imo, you could just have:
```py
formats.extend(self._extract_storyboard(vod_id, video.get('storyboard'), info.get('duration')))
```
and handle the None case in _extract_storyboard